ParabolInc/action
# chore(kudos): add kudos team settings

Fix #9130

We decided not to include UI in v1 

**How to test:**

- Run migration
- Run gql query, see kudos settings:

```
{
  viewer {
    teams {
      kudosEmoji
      giveKudosWithEmoji
    }
  }
}
```
